application/console/cards_console.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+class CardsConsole < ConsoleHelper
```

Comment:
Do not inherit a class from the helper. If the helper contains common helper methods, then include it as a module.

Reply:
created ConsoleHelper as module and included it in CardsConsole and Console classes.